### ISSUE TITLE
Feature: Set default table sort

### ISSUE THREAD

**Issue by obiwarn** [yogiben/meteor-admin#205]
e.g. default sort: by date desc -> always show the most recent entry on the first page

**Comment by nschulte** [comment]
+1

**Comment by HelloYie** [comment]
+1

**Comment by paniwani** [comment]
+1

**Comment by niranjans** [comment]
+1
I will try to see if I can get a PR done. 
Tabular has this functionality by passing an option like: `order: [[0, "desc"]]` to order with the first column.

**Comment by jeffyamada** [comment]
Did this ever get merged?

**Comment by niranjans** [comment]
Nope. I ended up using the customized local version of the package which is not a good solution.

**Comment by jeffyamada** [comment]
I was doing something similar for a while until it became more trouble than it was worth to keep a fork going, so instead I created a customized table template instead. If I get around to cleaning it up and making it easy for others to use it, I'll drop a line - but the big feature for the existing tables for me is how it manages subscriptions which is pretty important on larger collections.

**Comment by kenshinman** [comment]
Any fix for t his yet?

**Comment by kosset** [comment]
+1
